pyfstat/core.py

```diff
@@ -851,8 +851,6 @@ def estimate_min_max_CoverFreq(self):
 
     def get_fullycoherent_twoF(
         self,
-        tstart,
-        tend,
         F0,
```

Comment:
It might be a good further step to change this function to **all keyword args only**, so that there can be no confusion about argument order.

Reply:
Agree. Future work.